bach/bach/series/series_datetime.py

```diff
@@ -570,19 +567,106 @@ class SeriesTime(SeriesAbstractDateTime):
 
     * Postgres: utilizes the 'time without time zone' database type.
     * BigQuery: utilizes the 'TIME' database type.
+    * Athena: utilizes the 'double' database type.
```

Comment:
```suggestion
    * Athena: utilizes the 'double' database type. Storing the number of seconds since `00:00:00`.
```

I think without that information it would be hard to use this data-type in Bach with Athena data. People might not know how to convert their time columns to something that Bach understands. Of course number of seconds makes most sense if you have a double, but it could be a different format too like number of hours, or some silly format like `133712.345` to store `13:37:12.345`

Reply:
Here https://github.com/objectiv/objectiv-analytics/pull/1157/commits/6a8512be55348a1bc2655c9791423b43af26fc88